src/main/resources/db/migration/V1_68__oppfolgingsperiode_id.sql

```diff
@@ -0,0 +1,4 @@
+
+-- This is an inefficient way of storing UUID, but it should not be a problem at the current scale
+-- After migrating to postgres a proper column type can be used
+ALTER TABLE OPPFOLGINGSPERIODE ADD UUID CHAR(36);
```

Comment:
burde vi legge til en indeks og unik på denne med en gang.

Reply:
Indeks er mulig, men vet ikke om det er så mye poeng. Unik får man ikke lagt på siden det er flere null verdier i kolonnen

Reply:
vi kommer til og spøøe på uuiden etterhvert.
Slipper vi og legge til indeksen da

Reply:
Ser sånn ut ja.
`A UNIQUE constraint defines a set of columns that uniquely identify rows in a table only if all the key values are not NULL. If one or more key parts are NULL, duplicate keys are allowed.`

Tanken var forresten å legge på PK constraint etterpå, som vil sørge for non-null, unique og at det blir indeksert.